unittests/InstructionCountCI/VEX_map1.json

```diff
@@ -489,51 +489,42 @@
       ]
     },
     "vmovmskps rax, ymm0": {
-      "ExpectedInstructionCount": 41,
+      "ExpectedInstructionCount": 32,
```

Comment:
That's kind of cute. At some point we should do the same optimization that AVX128 does to make it better, but hard to care when nothing implements SVE with 256-bit wide registers.